tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -923,6 +926,23 @@ def ensure_descriptions_are_equal(
             ), f"Not equal on {path}{key}\nObtained={obtained_values} != {expected_values}"
             continue  # pragma no cover [bug on coverage.py
 
+        is_curve = isinstance(expected_value, Curve)
+        if is_curve:
+            obtained_curve = obtained_description_dict[key]
+            domain_unit = expected_value.GetDomain().GetUnit()
+            expected_domain_values = expected_value.GetDomain().GetValues(domain_unit)
+            obtained_domain_values = obtained_curve.GetDomain().GetValues(domain_unit)
+            assert np.allclose(
+                expected_domain_values, obtained_domain_values
+            ), f"Not equal on {path}{key}\nObtained={obtained_values} != {expected_values}"
+            image_unit = expected_value.GetImage().GetUnit()
+            expected_image_values = expected_value.GetImage().GetValues(image_unit)
+            obtained_image_values = obtained_curve.GetImage().GetValues(image_unit)
+            assert np.allclose(
+                expected_image_values, obtained_image_values
+            ), f"Not equal on {path}{key}\nObtained={obtained_values} != {expected_values}"
+            continue
```

Comment:
Don't forget to check this `Codecov` warning.

Reply:
This actually looks like a coverage bug. How is it possible line 944 to not be covered if other lines are?

Reply:
Note that other lines above are marked to skip coverage as well